lib/features/bendpoints/BendpointSnapping.js

```diff
@@ -7,7 +7,7 @@ import {
 var abs= Math.abs,
     round = Math.round;
 
-var TOLERANCE = 10;
+var TOLERANCE = 5;
```

Comment:
This is problematic. We shouldn't have to care if grid snapping exists or not.

Reply:
I think we should make bendpoint snapping totally optional and then remove it from bpmn-js. What's your opinion on that?

Reply:
I don't think we should deactivate bendpoint snapping. Instead we should use `setSnapped` to mark the event as snapped if bendpoint snapping decides to snap it so it won't be snapped by grid snapping again.

If we decativate bendpoint snapping you can't snap to existing bendpoints with higher priority if they are not on the grid:

![16-05-_2019_16-41-01](https://user-images.githubusercontent.com/7633572/57862919-6feee500-77f9-11e9-8dcc-327b5bc7b729.gif)

Reply:
Cool. I'm reverting the commit about separation of features.